install-deps.sh

```diff
@@ -116,8 +116,15 @@ else
     case $ID in
     debian|ubuntu|devuan)
         echo "Using apt-get to install dependencies"
+        # Depdencies for Apache Arrow
+        curl https://dist.apache.org/repos/dist/dev/arrow/KEYS | sudo apt-key add -
```

Comment:
curl is not installed until line #127 below, so please move that around in the minimal way if possible

Reply:
Done